dev-haskell/beam-core/beam-core-0.9.0.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Type-safe, feature-complete SQL query and manipulation interface for Haskell"
+HOMEPAGE="http://travis.athougies.net/projects/beam.html"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="werror"
+
+RDEPEND=">=dev-haskell/aeson-0.11:=[profile?] <dev-haskell/aeson-1.5:=[profile?]
+	>=dev-haskell/dlist-0.7.1.2:=[profile?] <dev-haskell/dlist-0.9:=[profile?]
```

Comment:
Hrm, I also have `dlist-1.0` in #1118.

@trofi should @hololeap just not worry about the newer versions, and keep the PR as-is?

Reply:
Yeah, if this PR is mostly ready and #1118 is not I'd give this PR a favour and merge it.